src/maxwell-notes.md

```diff
@@ -11,7 +11,7 @@ With electric current density, ${\bf J}$, magnetic current density,
 ${\bf M}$, electric conductivity, $\overline{\sigma}$, magnetic
 conductivity, $\overline{\sigma}_M$, and electric charge density,
 $\rho$. We will sometimes refer to these equations by the names Ampère's
-Law, Faraday's Law, Guass' Law, and the Transversality Condition
+Law, Faraday's Law, Gauss's Law, and the Transversality Condition
```

Comment:
We need to fix the `Guass` misspelling, but I believe either `Gauss' Law` or `Gauss's Law` is fine here.

I personally prefer `Gauss' Law`.

Reply:
Hi Tzanio,

I'm happy to revert the change, but note that every other document on the website refers to it as `Gauss's Law`.
I'll let you make the final call.